libs/domain/checkout/services/src/adapter/mock/mock-checkout.ts

```diff
@@ -0,0 +1,308 @@
+import { PlaceOrderData } from '@spryker-oryx/checkout';
```

Comment:
Could you please add a comment at the top on how to use these files.
Smth like "to be replaced with a backend system API"